src/Microsoft.Tye.Hosting/Model/IngressRule.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Tye.Hosting.Model
+{
+    public class IngressRule
+    {
+        public IngressRule(string? host, string? path, string service)
+        {
+            Host = host;
```

Comment:
Do we want the dashboard API to return these rules in api/v1/services?

Reply:
Yea, I haven't decided if it should differ from services. Right now it'll show IngressStatus, I could add the rules there but it's not really important at the moment.